mltsp/util.py

```diff
@@ -81,3 +82,44 @@ def is_running_in_docker():
     else:
         in_docker_container = False
     return in_docker_container
+
+
+def cast_model_params(model_type, model_params):
+    """Cast model parameter strings to expected types."""
+    from .ext.sklearn_models import model_descriptions
+    for entry in model_descriptions:
+        if entry["abbr"] == model_type:
+            params_list = entry["params"]
+            break
+    for k, v in model_params.items():
+        if v == "None":
```

Comment:
Should we accept "" as `None` also? I was thinking at the front-end that an empty text box might look nicer than one with "None", would make it easier to just scan visually and see what params are being set...